schema.sql

```diff
@@ -35,7 +37,11 @@ create table customers (
   -- UUID from auth.users
   id uuid references auth.users not null primary key,
   -- The user's customer ID in Stripe. User must not be able to update this.
-  stripe_customer_id text
+  stripe_customer_id text,
+  -- The customer's billing address, stored in JSON format
+  billing_address jsonb,
+  -- Stores your customer's payment instruments.
+  payment_method jsonb
```

Comment:
https://stripe.com/docs/api/payment_methods

Should this be a list? i.e. `payment_methods`. Looks like there can be several methods/cards

Reply:
there's only one default_payment method, that's what we'll use here 👍